src/frontend/package.json

```diff
@@ -33,20 +33,20 @@
     "sass": "1.67.0"
   },
   "devDependencies": {
-    "@babel/core": "7.22.15",
-    "@babel/preset-env": "7.22.15",
+    "@babel/core": "7.22.20",
+    "@babel/preset-env": "7.22.20",
     "@babel/register": "7.22.15",
-    "@types/react": "18.2.21",
+    "@types/react": "18.2.22",
     "@types/react-datepicker": "4.15.0",
     "@types/react-dom": "18.2.7",
     "@typescript-eslint/eslint-plugin": "5.40.0",
     "@typescript-eslint/parser": "5.40.0",
     "@vitejs/plugin-react": "2.1.0",
-    "@wdio/cli": "8.16.4",
-    "@wdio/cucumber-framework": "8.16.4",
-    "@wdio/local-runner": "8.16.4",
-    "@wdio/mocha-framework": "8.16.3",
-    "@wdio/spec-reporter": "8.16.3",
+    "@wdio/cli": "8.16.11",
+    "@wdio/cucumber-framework": "8.16.11",
+    "@wdio/local-runner": "8.16.11",
+    "@wdio/mocha-framework": "8.16.11",
```

Comment:
We actually aren't using this so you can remove it

Reply:
> We actually aren't using this so you can remove it

do you think it is right we remove the whole dependencies for webdriver if we decide to change to another testing framework? 
`"@wdio/cli": "8.16.11",
  "@wdio/cucumber-framework": "8.16.11",
  "@wdio/local-runner": "8.16.11",
  "@wdio/mocha-framework": "8.16.11",
  "@wdio/spec-reporter": "8.16.9",
  "geckodriver": "4.2.1",
  "wdio-cucumberjs-json-reporter": "5.1.7",
  "wdio-geckodriver-service": "5.0.2",
  "wdio-wait-for": "3.0.7"`

Reply:
I dont think we should yet. Lets wait until we try out Play Wright